packages/tabs/src/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Flex, Text } from 'pcln-design-system'
+import styled from 'styled-components'
+
+const TabTitle = styled(Text)`
+  border-bottom: ${props =>
+    props.isActive ? `4px solid ${props.activeColor}` : 'none'};
+  cursor: pointer;
+`
+
+class Tabs extends React.Component {
```

Comment:
This can be a pure component: https://codeburst.io/when-to-use-component-or-purecomponent-a60cfad01a81

The approach I have taken with component libraries in the past is that you use `PureComponent` at least to begin with and have people adjust their data to work with it. If we need to downgrade it to a regular component then that is not a breaking change for anyone. However, this is not true for the reverse, hence why it is better to start with `PureComponent`.

Do people agree with this approach? I haven't seen much use of `PureComponent` around but it is vitally important to get the full benefits of React.